userguide/tutorials/database_migrations.adoc

```diff
@@ -70,7 +69,7 @@ For example, to download all migration files for Kill Bill between 0.16.12 and 0
 kpm migrations killbill killbill-0.16.12 killbill-0.18.0
```

Comment:
Does anything need to be done to get rid of the above alert?

Reply:
Hi @reshmabidikar I don't think so. I just removed couple of words in the text, rest all formatting is the same.

Reply:
Looking into the [details ](https://github.com/killbill/killbill-docs/security/code-scanning/5248) of this alert looks like the code needs to be formatted using `[source, bash]`. I'll merge the PR for now, but maybe this can be addressed in a subsequent PR?